SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -18,14 +18,14 @@ void ARegisterAutoDestroyActorsTestPart1::PrepareTest()
 	{ // Step 1 - Spawn Actor On Auth
 		AddStep(TEXT("SERVER_1_Spawn"), FWorkerDefinition::Server(1), nullptr, [this]() {
 			UWorld* World = GetWorld();
-			int NumVirtualWorkers = GetNumberOfServerWorkers();
+			int32 NumVirtualWorkers = GetNumberOfServerWorkers();
 
 			// spawn 1 per server worker
 			// since the information about positioning of the virtual workers is currently hidden, will assume they are all around zero
 			// and spawn them in that radius
 			FVector SpawnPosition = FVector(200.0f, -200.0f, 0.0f);
```

Comment:
nit - what do these magic numbers mean?

Reply:
added a comment, but it's basically just for visuals, not too important - the more important part is the rotation, which is explained